hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -241,32 +237,28 @@ public RetryAction shouldRetry(Exception exception, int retries,
           throws Exception {
```

Comment:
It has failOvers parameter, I see it has been removed the top, any reason for not using this?

Reply:
In case of NotLeaderException, the failover count will not increase as we failover manually and give RetryAction=Retry. In the other case, the retry and failover count will always be equal as the action is FailoverAndRetry.